packages/requestNetworkSmartContracts/contracts/synchrone/RequestEthereum.sol

```diff
@@ -557,9 +539,9 @@ contract RequestEthereum is RequestEthereumCollect {
 
 	/*
 	 * @dev Function internal to manage fund mouvement
-	 * @dev We had to chose between a withdraw pattern, a send pattern or a send + withdraw pattern and chose the last. 
-	 * @dev The withdraw pattern would have been a too big inconvenient for the UX. The send pattern would have allowed someone to lock a request. 
-	 * @dev The send + withdraw pattern will have to be clearly explained to users. If the payee is a contract which can let a transfer fail, it will need to be able to call a withdraw function from Request. 
+	 * @dev We had to chose between a withdraw pattern, a transfer pattern or a transfer+withdraw pattern and chose the transfer pattern. 
```

Comment:
Suggestion:
We had to chose between a withdrawal pattern, a transfer pattern or a transfer+withdrawal pattern and chose the transfer pattern. 
The withdrawal pattern would make UX difficult. The transfer+withdrawal pattern would make contracts interacting with the request protocol complex.
N.B.: The transfer pattern will have to be clearly explained to users. It enables a payee to create unpayable requests.

Reply:
updated